ui/app/components/variable-form.js

```diff
@@ -207,6 +208,7 @@ export default class VariableFormComponent extends Component {
    * @param {KeyboardEvent} e
    */
   @action setModelPath(e) {
+    set(this, 'path', e.target.value);
```

Comment:
This, and the `entry` set above it, are side-effects of the Helios input component not doing two-way data binding. For some additional context, two-way data binding is kind of a bygone convention in most JS frameworks at this point so it's not that weird, but we were using that bygone convention for quite awhile. Now, we are explicit about "When you do something that indicates needing to save something, save it".

In this case, that happens in the template on keystroke (`{{on "input" (action this.setModelPath)}}`), but it's also common to do this on element blur, on form submit, etc. We're just eager beavers here.